scripts/mkcloud

```diff
@@ -394,18 +394,19 @@ function onhost_create_cloud_lvm()
     fi
 
     echo "Checking for LVs treated by LVM as valid PV devices ..."
-    if lvmdiskscan | egrep "/dev/($cloud/|mapper/$cloud-)"; then
+    if lvmdiskscan | egrep "/dev/($cloudvg/|mapper/$cloudvg-)"; then
         error=$(cat <<EOF
 Error: your lvm.conf is not filtering out mkcloud LVs.
 Please fix by adding the following regular expressions
 to the filter value in the devices { } block within your
 /etc/lvm/lvm.conf file:
 
-    "r|/dev/mapper/$cloud.*|", "r|/dev/$cloud/.*|"
+    "r|/dev/mapper/$cloudvg-|", "r|/dev/$cloudvg/|", "r|/dev/disk/by-id/|"
 
-The filter should also include something like "r|/dev/dm-1[56]|", but
+The filter should also include something like "r|/dev/dm-|" or "r|/dev/dm-1[56]|", but
```

Comment:
Similarly to `/dev/disk/by-id/`, I'm worried that filtering `/dev/dm-` would exclude other important devices.